Added citation service info (#181)
WCRP-CMIP/cmip7-guidance: docs/CMIP7/Citation_Guidance.md

---
layout: default
title: CMIP7 Citation Guidance
---

# Guidance for CMIP7 Citations

> The CMIP7 Citation service is hosted at [https://cmip7-citations.ceda.ac.uk/citations/](https://cmip7-citations.ceda.ac.uk/citations/).

Reviewer: ```suggestion
The CMIP7 Citation service is hosted at <https://cmip7-citations.ceda.ac.uk/citations/>.
```

> CMIP7 data citations will be created automatically in response to data publication on ESGF. The granularity of the data citations will be at the level of a model's contribution to an experiment.

Reviewer: ```suggestion
CMIP7 data citations will be created automatically in response to data publication on ESGF. The granularity of the data citations will be at the *experiment* level: a data citation is created for each experiment contributed by a given model.
```
Suggesting the above phrasing might be clearer (open to other suggestions too).

In the [user guidance section](https://wcrp-cmip.github.io/cmip7-guidance/docs/CMIP7/Guidance_for_users/#2-terms-of-use-and-citations-requirements) we say that citations are created at both the experiment and activity level, where the activity one refers to all experiments defined by that activity (since this was the CMIP6 practice). It also recommends using the experiment citation level whenever possible (see the "Do I cite the model based on activity or experiment?" call-out box). Is this true - is the citation service still defining citations at the activity level as well? (If this is said elsewhere and I missed it, apologies.) We should make these two parts of the guidance pages consistent with each other.

Author: @JamesAnstey the citation service is now only making citations at the experiment-level, there are no activity level citations planned, although the design of the service does not preclude this use case. The description "model's contribution to an experiment" appears in multiple other places including the reporting of the citation service internally to stakeholders.

> CMIP7 data providers will need to review the information on the data citation landing pages and add party information before DOIs can be issued. An [Approver Guide Document](https://github.com/cedadev/cmip7-citation-service/blob/main/docs/CitationReviewerGuide.pdf) has been prepared to articulate how to use the UI to make edits and publish citation information (i.e Mint DOIs)

Reviewer: That doc looks really useful - I wonder if (similar to Matt's comment about the png) a copy could be included here in the guidance repo?

Author: I can add copies of both documents to this repository yes, I think that would be best for now although there may need to be PRs for changes later on.

The Citation Service is configured to natively support CMIP7, CORDEX-CMIP6 and CMIP6Plus via the [ESGVOC Python Package](https://esgf.github.io/esgf-vocab/).

> !!! tip "A Note on CMIP6 DCPP sub-experiments "

Reviewer: ```suggestion
!!! tip "A Note on CMIP6Plus DCPP sub-experiments "
```
since CMIP6Plus is what the new citation service supports? (Or perhaps: "CMIP6 / CMIP6Plus DCPP sub-experiments"?)

Author: I will bring these up to others in the Citation group to check but this sounds sensible to me.

    Sub-experiments are not part of the data model for the CMIP7 citation service, however the sub-experiment-id can still be manually included in the citation title field. Only the core CMIP7 search facets are validated against the ESGVOC API, so the sub-experiment ID will not appear in the search facets area, but they will be findable via the fuzzy search in the free text search box.

> ![Citation Flow Diagram](https://raw.githubusercontent.com/cedadev/cmip7-citation-service/main/docs/CitationServiceFlowDiagram.png)

Reviewer 2: If it doesn't cause any issues at the CEDA end I would be tempted to upload this here (safer in case you re-arrange the citation service repo without realising that the docs break).  Alternatively point at a tagged version of the repo rather than the main branch.

Above is the flow diagram showing internal processes within the citation service, including the user-facing landing pages (Purple Boxes).

> Additional information for developers can be found at the citation service documentation site [https://cedadev.github.io/cmip7-citation-service](https://cedadev.github.io/cmip7-citation-service)

Reviewer: ```suggestion
Additional information for developers can be found at the citation service documentation site: <https://cedadev.github.io/cmip7-citation-service>
```
